pkg/tangy/interface.go

```diff
@@ -0,0 +1,55 @@
+package tangy
+
+import (
+	"context"
+	"fmt"
+
+	zerologadapter "github.com/jackc/pgx-zerolog"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/jackc/pgx/v5/tracelog"
+	"github.com/rs/zerolog/log"
+)
+
+func New(dbConfig Database, logConfig Logger) (Tangy, error) {
+	pxConfig, err := pgxpool.ParseConfig(dbConfig.Url())
+	if err != nil {
+		return nil, err
+	}
+
+	if dbConfig.PoolLimit == 0 {
+		dbConfig.PoolLimit = DefaultMaxPoolLimit
+	}
+	pxConfig.MaxConns = int32(dbConfig.PoolLimit)
```

Comment:
we should probably have a method  that would allow the application to close/release the connection pool